backend/src/exercise/patient-ticking.ts

```diff
@@ -46,6 +50,11 @@ export function patientTick(
             // Only look at patients that are alive and have a position, i.e. are not in a vehicle
             .filter((patient) => isAlive(patient.health) && patient.position)
             .map((patient) => {
+                // update the time a patient is being treated, to check for pretriage later
+                let treatmentTime = patient.treatmentTime;
+                if (patient.isBeingTreated) {
+                    treatmentTime += patientTickInterval;
+                }
```

Comment:
```suggestion
                const treatmentTime = patient.isBeingTreated
                    ? patient.treatmentTime
                    : patient.treatmentTime + patientTickInterval;
```

let makes the code more complicated to read because status could potentially be reassigned anywhere in the next 50 lines...

Reply:
The other way around... XD